plugins/modules/cmci_create.py

```diff
@@ -214,6 +214,239 @@
       description: The URL used for the request.
       returned: success
       type: str
+feedback:
```

Comment:
I wonder if we could use a python variable for this doc rather than repeating it.  Don't see why not, but it might not work.  Give it a try!